src/stores/tradeListStore.ts

```diff
@@ -2,7 +2,7 @@ import {computed, observable, runInAction} from 'mobx';
 import {WampApi} from '../api';
```

Comment:
could you plz rename the file to _tradeStore.ts_ and underlying `TradeListStore` class to `TradeStore`